tutorials/README.md

```diff
@@ -16,47 +16,81 @@ Check that your tutorial renders properly.
-there. Make sure to update the title in the notebook too.
+The learning platform ignores the top-level heading of the notebook. These
+headings are only included in the notebook for writers' convenience.
+
+> [!CAUTION]
+> The title dictates the URL slug. Do not change a tutorial's title without
+> having appropriate redirects set up.
+
+If you want to change the title of a notebook, find the page in the CMS and
+change its title there. Make sure to update the title in the notebook too.
 
 ## Adding new tutorials
 
 Each tutorial has its own folder in `tutorials/`. Within that folder is the content notebook
 (ending in `.ipynb`) and an optional `images` folder.
 
-To add a new tutorial to the learning platform, go to
-https://learning-api.quantum.ibm.com/admin/ and choose "Create item" (the blue
-`+` in the top-right corner). Enter all the information, but leave the content
-field blank. This will create the tutorial in "draft" mode.
+Once you've decided to add a new tutorial to the learning platform, a person
+with appropriate permissions to the learning platform should follow these
+steps.
 
-Once you've added the tutorial, copy the tutorial's URL relative to `content`
-(that is, the URL after `https://learning-api.quantum.ibm.com/admin/content/`).
-The copied URL should be of the form `tutorials/<UUID>`. Next, make a new entry
-in the `learning-api.conf.yaml` file in this folder. Set the `path` attribute
-to the name of the tutorial folder in this repo, and `urlProduction` to the
-tutorial ID you copied earlier. Repeat the same process for the staging
-environment (https://learning-api-dev.quantum.ibm.com/admin/).
+1. **Create a new page on the learning platform**
 
-Next, you will need to add the tutorial to our API token permissions. To do
-that, contact Abdón Rodríguez and ask to add the tutorial IDs to the
-Qiskit/documentation API keys.
+   For both environments (staging and production), log on to the CMS and go to
+   the "tutorials" tab. Click "Create item" (the blue `+` in the top-right
+   corner) and enter all the information, but leave the content field blank.
+   This will create the tutorial in "draft" mode.
 
-To push content from this repo to the learning platform, run the [Deploy
-tutorials](https://github.com/Qiskit/documentation/actions/workflows/deploy-tutorials.yml)
-workflow. This will push the content of **all** tutorials from the main branch.
-Once your content is uploaded, you can go back to the admin panel for that
-tutorial and publish the lesson to take it out of draft mode.
+2. **Give our token permission to push to the tutorial**
+
+   > TODO: Improve this; I don't have permission to do this so can't give more details.
+
+   Next, we need to give the token in this repo permission to push to the new
+   tutorial. In the CMS, go to "Users", then give "Token: Qiskit/documentation"
+   permission to push to the new tutorial.
+
+3. **Link the notebook in this repo to the learning platform**
```

Comment:
I think this section needs to make clear you should open a followup PR to do this